Sketch.java

```diff
@@ -24,13 +24,59 @@ public void setup() {
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	drawFirstFace(300,300);
+  drawSecondFace(250, 60, 255, 192, 203, 0, 255);
+  getParameters(width/2, height/2); 
```

Comment:
this method draws something, however, serves no purpose as a method that returns an int.